client/components/domains/domain-registration-suggestion/index.jsx

```diff
@@ -217,6 +219,11 @@ class DomainRegistrationSuggestion extends React.Component {
 			suggestion: { domain_name: domain, product_slug: productSlug, cost },
 		} = this.props;
 
+		let buttonStyles = { primary: true };
+		if ( abtest( 'domainSearchButtonStyles' ) === 'onePrimary' ) {
+			buttonStyles = ! isFeatured ? {} : this.props.buttonStyles;
```

Comment:
If we go by the comment below, it's probably a good idea to just reset styles here if not featured.

Reply:
Disregard, missed the `Secondary` featured.